### PR TITLE
More bem and fixes

### PR DESCRIPTION
**Pull request by therealcoder1337**

### CODE REVIEW

File: src/lib/components/toggle/QToggle.svelte [hunk 1]
```diff
@@ -11,14 +11,14 @@
     userClasses: QToggleProps["userClasses"] = undefined;
   export { userClasses as class };
 
-  $: classes = createClasses([
-    "q-toggle",
-    leftLabel && "reverse",
-    disable && "disable",
+  $: classes = createClasses([leftLabel && "reverse", disable && "disable", userClasses], {
```
Comment (KilDesu) [inline src/lib/components/toggle/QToggle.svelte:14]: Should be "reversed" and "disabled"

File: src/lib/css/mixins/selection.scss [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+  cursor: pointer;
+  display: inline-flex;
+  align-items: center;
+
+  @if $type == checkbox or $type == radio {
+    > input {
+      width: 1.5rem;
+      height: 1.5rem;
+      opacity: 0;
+      background: red !important;
```
Comment (KilDesu) [inline src/lib/css/mixins/selection.scss:16]: Why this?

File: src/lib/css/mixins/selection.scss [hunk 3]
```diff
@@ -0,0 +1,185 @@
...
+    margin: 0 auto;
+    outline: none;
+    color: var(--primary);
+    position: absolute;
+    left: -1.5rem;
+    background-color: transparent;
+    border-radius: 50%;
+    user-select: none;
+    z-index: 1;
+    box-shadow: 0 0 0 0 var(--active);
```
Comment (KilDesu) [inline src/lib/css/mixins/selection.scss:72]: Shouldn't it be box-shadow: none?

File: src/lib/css/mixins/selection.scss [hunk 4]
```diff
@@ -0,0 +1,185 @@
...
+      content: "radio_button_unchecked";
+    }
+
+    > input:checked + span::before {
+      content: "radio_button_checked";
+    }
+  }
+
+  &--icon > span::before {
+    content: "" !important;
```
Comment (KilDesu) [inline src/lib/css/mixins/selection.scss:161]: It's the important really necessary here?

**Review by KilDesu — COMMENTED** [review]
Good job!

**Review by KilDesu — APPROVED** [review]

File: src/lib/css/mixins/selection.scss (hunk 2 above)
Comment (therealcoder1337) [reply src/lib/css/mixins/selection.scss:16]: Thanks, probably left over from debugging.

File: src/lib/css/mixins/selection.scss (hunk 3 above)
Comment (therealcoder1337) [reply src/lib/css/mixins/selection.scss:72]: No, it's straight from Beer CSS.

File: src/lib/css/mixins/selection.scss (hunk 4 above)
Comment (therealcoder1337) [reply src/lib/css/mixins/selection.scss:161]: Hmm, good question. But I didn't explicitly add it myself, so I'm not going to change it for now. Just a precaution to not break any styles.

**Pull request merged**